cfgov/preprocessed/css/organisms/sidebar-breakout.less

```diff
@@ -0,0 +1,79 @@
+/* topdoc
+  name: Sidebar Breakout
+  family: cfgov-organism
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-sidebar-breakout">
+            <h3 class="o-sidebar-breakout_heading">
+            </h3>
+            <div class="o-sidebar-breakout_column">
+                <div class="u-flexible-container">
+                    <div class="u-flexible-container_inner"
+                         style="background-image:url();">
+                    </div>
+                </div>
+            </div>
+            <div class="o-sidebar-breakout_column">
+                <h4 class="o-sidebar-breakout_intro-heading">
+                </h4>
+                <p></p>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-sidebar-breakout
+            .o-sidebar-breakout_heading
+            .o-sidebar-breakout_column
+              .u-flexible-container
+                .u-flexible-container_inner
+            .o-sidebar-breakout_column
+              .o-sidebar-breakout_intro-heading
+  tags:
+    - cfgov-molecules
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+
+.o-sidebar-breakout {
+    border-top: 1px solid @gray-50;
```

Comment:
Just noting that this should be updated to use the rule-break atom when https://github.com/cfpb/cfgov-refresh/commit/c295067805af9db04def644621532424d1bdbe4e is merged.